### PR TITLE
Adding GCP COS base image shutdown scripts to GCP k8s release bundle

### PR DESCRIPTION
**Pull request by vishh**
Depends on https://github.com/kubernetes/kubernetes/pull/64294

### CODE REVIEW

**Comment by dims** [comment]
We still need this right? @listx @vishh @david-mcmahon

**Comment by vishh** [comment]
@dims yes we do. I'm not sure who is going to merge this.

**Comment by vishh** [comment]
@krzyzacy can you review this PR?

File: lib/releaselib.sh [hunk 1]
```diff
@@ -631,6 +631,8 @@ release::gcs::locally_stage_release_artifacts() {
       || return 1
     release::gcs::stage_and_hash $gcs_stage $gci_path/configure.sh extra/gce \
       || return 1
+    release::gcs::stage_and_hash $gcs_stage $gci_path/shutdown.sh extra/gce \
```
Comment (ixdy) [inline lib/releaselib.sh]: won't this cause problems on older branches where this file doesn't exist?

**Comment by listx** [comment]
Sorry, I just learned that I was assigned this PR.

It looks like @vishh authored the other patch in kubernetes/kubernetes#64294 which got merged already. Is https://github.com/kubernetes/kubernetes/pull/64294#issuecomment-393916199 a blocker for this?

File: lib/releaselib.sh (hunk 1 above)
Comment (vishh) [reply lib/releaselib.sh]: Is there a versioning story for this lib? How do you recommend I add this extra dependency?

**Comment by vishh** [comment]
@listx https://github.com/kubernetes/kubernetes/pull/64294#issuecomment-393916199 is not a blocker.

**Comment by krzyzacy** [comment]
(I have no context here, I'll punt to @ixdy and @listx )
cc @yguo0905 who might also be interested

File: lib/releaselib.sh (hunk 1 above)
Comment (ixdy) [reply lib/releaselib.sh]: I don't think there is one, but maybe add a conditional to see if the file exists first? see e.g. L613-618

Comment (ixdy) [reply lib/releaselib.sh]: L620-624 is another good example

Comment (vishh) [reply lib/releaselib.sh]: @ixdy updated code. PTAL

**Comment by vishh** [comment]
@ixdy PTAL

**Pull request merged**